name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/RaiffeisenBankgruppePDFExtractor.java

```diff
@@ -677,8 +749,8 @@ private void addAccountStatementTransactions()
 
                     if (context.get("nr").compareTo("01") == 0  && Integer.parseInt(v.get("month")) < 3)
                     {
-                        Integer year = Integer.parseInt(context.get("year")) + 1;
-                        t.setDateTime(asDate(v.get("day") + "." + v.get("month") + "." + year.toString()));
+                        int year = Integer.parseInt(context.get("year")) + 1;
+                        t.setDateTime(asDate(v.get("day") + "." + v.get("month") + "." + Integer.toString(year)));
```

Comment:
If you already have day, month, and year - there is no need to convert them to a string and parse them again:

```
LocalDate.of(year, month, day).atMidnight()
```